### PR TITLE
[WIP] Homepage Fixes

### PR DESCRIPTION
**Pull request by shkamaru**

### CODE REVIEW

File: pontoon/homepage/static/css/homepage.css [hunk 1]
```diff
@@ -316,10 +312,12 @@ h2 {
 
 #section-6 .footer .github {
   text-align: center;
+  font-size: 14px;
 }
 
 #section-6 .footer .contact {
   text-align: right;
+  font-size: 14px;
```
Comment (mathjazz) [inline pontoon/homepage/static/css/homepage.css]: It would be simpler and easier to maintain if you added this property to `#section-6 .footer`.

**Comment by mathjazz** [comment]
Can you please also remove the `pontoon.homepage.migrations` folder?

**Comment by mathjazz** [comment]
@shkamaru Hey, what's the status of this PR?

**Comment by shkamaru** [comment]
Nothing new is done. We can merge these changes because I won't be able to
continue on the animations now.

**Comment by mathjazz** [comment]
OK. Could you address https://github.com/mozilla/pontoon/pull/1068/files#r213270074 please?

**Pull request merged**